extensions/gitpod/src/open_repo_context.tsx

```diff
@@ -0,0 +1,275 @@
+    return (
+      <List
+        searchBarPlaceholder="Filter `/b` for branches, `/p` for Pull Request, `/i` for issues"
+        onSearchTextChange={parseSearchOptions}
+        navigationTitle={"Add `/me` to filter from your profile 🧡"}
+        throttle
+      >
+        {sections.includes("/b") && JSON.parse(cache.get(repository.name)!)?.branches !== undefined && (
+          <List.Section
+            key={"Branches"}
+            title={"Branches"}
+            subtitle={pluralize(JSON.parse(cache.get(repository.name)!)?.branches.length, "Branch", {
+              withNumber: true,
+            })}
+          >
+            {JSON.parse(cache.get(repository.name)!).branches.map((branch: BranchDetailsFragment) => {
+              return (
+                <BranchListItem
+                  key={branch.branchName}
+                  mainBranch={repository.defaultBranchRef?.defaultBranch ?? ""}
+                  repository={forAuthor ? `${viewer?.login}/${repository.name}` : repository.nameWithOwner}
+                  branch={branch}
+                  viewer={viewer}
+                />
+              );
+            })}
+          </List.Section>
+        )}
+
+        {sections.includes("/p") && JSON.parse(cache.get(repository.name)!)?.pullRequest !== undefined && (
+          <List.Section
+            key={"Pulls"}
+            title={"Pull Requests"}
+            subtitle={pluralize(JSON.parse(cache.get(repository.name)!)?.pullRequest.length, "Pull Request", {
+              withNumber: true,
+            })}
+          >
+            {JSON.parse(cache.get(repository.name)!).pullRequest.map((pullRequest: PullRequestFieldsFragment) => {
+              if (pullRequest.closed && pullRequest.merged) {
+                return <PullRequestListItem key={pullRequest.id} pullRequest={pullRequest} viewer={viewer} />;
+              } else if (!pullRequest.closed) {
+                return <PullRequestListItem key={pullRequest.id} pullRequest={pullRequest} viewer={viewer} />;
+              }
+            })}
+          </List.Section>
+        )}
+
+        {sections.includes("/i") && JSON.parse(cache.get(repository.name)!)?.issues !== undefined && (
+          <List.Section
+            key={"Issues"}
+            title={"Issues"}
+            subtitle={pluralize(JSON.parse(cache.get(repository.name)!)?.issues.length, "Issue", { withNumber: true })}
+          >
+            {JSON.parse(cache.get(repository.name)!).issues.map((issue: IssueFieldsFragment) => {
+              return <IssueListItem key={issue.id} issue={issue} viewer={viewer} />;
+            })}
+          </List.Section>
+        )}
+      </List>
+    );
+  }
+
+  return (
+    <List
+      isLoading={isPRLoading}
+      searchBarPlaceholder="Filter `/b` for branches, `/p` for Pull Request, `/i` for issues"
+      onSearchTextChange={parseSearchOptions}
+      navigationTitle={"Add `/me` to filter from your profile 🧡"}
```

Comment:
Thank You So much for your response! 🍊

Basically when the repo opens up in raycast, when you enter the phrase `/me` it shows you the context related to you, like your PRs, issues and branches, so that you get a faster access around things, which you were working on for the same repo, by default, it shows you generic context.

Uploading Screen Recording 2023-01-20 at 10.15.00 PM.mov…

As you can see in the above video, on applying `/me` I switch the context to show my stuff.

Reply:
You can read more about [navigationTitle here](https://developers.raycast.com/basics/prepare-an-extension-for-store#navigation-title) - it's not meant for small messages - I still think we need to have the context of the {REPO} instead somehow

Reply:
Sure thing @pernielsentikaer, I have made the necessary changes
Thank you so much

Reply:
Hey @pernielsentikaer, would love your review again! Can't wait to get this in the store 🚀🚀